tests/UI/pages/BO/advancedParameters/team/profiles/index.js

```diff
@@ -72,7 +72,7 @@ module.exports = class Profiles extends BOBasePage {
 
   /**
    * Reset input filters
-   * @return {Promise<textContent>}
+   * @returns {Promise<integer>}
```

Comment:
why sometimes int, and sometimes integer? :sweat_smile:

Reply:
Thank you :D I will check all returns ;)